packages/react-components/src/components/Avatar/Avatar.helpers.ts

```diff
@@ -18,12 +18,27 @@ export function getBackgroundColor(initials: string): string | undefined {
 }
 
 export function getInitials(name = '', count = 2): string {
-  return name
-    .split(' ')
-    .map((el) => el.charAt(0))
-    .join('')
-    .substring(0, count)
-    .toUpperCase();
+  const nameArray = name.split(' ');
```

Comment:
To deal with multiple spaces, e.g. `foo    bar`:
```suggestion
  const nameArray = name.split(/\s+/);
```